simulator/src/main/java/com/hazelcast/simulator/coordinator/PerformanceStateContainer.java

```diff
@@ -86,9 +86,9 @@ public String formatPerformanceNumbers(String testCaseId) {
             return "";
         }
         String latencyUnit = "µs";
-        long avgLatencyValue = round(performanceState.getIntervalAvgLatency());
-        long percentileLatencyValue = performanceState.getIntervalPercentileLatency();
-        long maxLatencyValue = performanceState.getIntervalMaxLatency();
+        long avgLatencyValue = round(performanceState.getIntervalAvgLatency() / MICROSECONDS.toNanos(1));
```

Comment:
This looks very confusing now. Also the latency values should be rounded, and now you are dividing them again. Shouldn't this be `NANOS.toMicros(round(...));`?